### PR TITLE
3.2.x updates

### PR DESCRIPTION
**Pull request by davidkelly**

### CODE REVIEW

File: src/test/java/org/springframework/data/couchbase/repository/N1qlCouchbaseRepositoryIntegrationTests.java [hunk 1]
```diff
@@ -90,6 +96,40 @@ public void cleanUp() {
     try { partyRepository.deleteById(KEY_PARTY); } catch (DataRetrievalFailureException e) {}
   }
 
+  @Test
+  public void shouldBeThreadsafe() {
+    // This doesn't guarantee it, but we should catch most thread issues without
+    // taking too long here...
+    int runs = 50;
+    for (int i=0; i<runs; i++) {
```
Comment (mp911de) [inline src/test/java/org/springframework/data/couchbase/repository/N1qlCouchbaseRepositoryIntegrationTests.java]: Nit: Reformat code.

**Comment by davidkelly** [comment]
So -- @mp911de should I just close this, and cherry pick directly into the 3.2.x branch?  I'm happy to do it -- just want to be sure I didn't misunderstand.  thx!

**Comment by mp911de** [comment]
Yes, exactly.

**Comment by davidkelly** [comment]
thx!  Will do today!

**Pull request closed without merging**